impress-org/givewp
# Donors are now able to log into Donor Dashboard on hosts with page caching

Resolves #5764 

## Description

This PR resolved the issues faced on hosts with aggressive page caching by ensuring that the WP API nonce is not required in order to access the donor dashboard, and ensuring that query params are passed to the iframe which contains the Donor Dashboard application.

## Affects

This PR affects some API query logic for the Donor Dashboard frontend, and some iframe markup output logic in the backend.

## Visuals

N/A

## Testing Instructions

On a WP Engine install:
1. Make a donation with an email not associated with a WP acct
2. Go to Donor Dashboard
3. Request email verification
4. See that the email is sent successfully, and that you can access the Donor Dashboard using the link in the email
5. 
## Pre-review Checklist
